src/main/java/Calculator.java

```diff
@@ -0,0 +1,97 @@
+import java.util.ArrayList;
+import java.text.DecimalFormat;
+import java.util.Scanner;
+
+class Product {
+    private String name;
+    private double price;
+
+    public Product(String name, double price) {
+        this.name = name;
+        this.price = price;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public double getPrice() {
+        return price;
+    }
+
+    public String toString() {
+        return name + " - " + new DecimalFormat("0.00").format(price) + " рубля";
+    }
+}
+
+class Calculator {
+    private int persons;
```

Comment:
🍏 Так как в коде класса эта поле не изменяется после инициализации, ему можно добавить модификатор final 
```suggestion
    private final int persons;
```